dd-sdk-android-gradle-plugin/src/test/kotlin/com/datadog/gradle/plugin/internal/GitRepositoryDetectorTest.kt

```diff
@@ -109,25 +107,42 @@ internal class GitRepositoryDetectorTest {
             ProcessBuilder("git", "init")
                 .directory(tempDir)
                 .start()
-                .waitFor(5, TimeUnit.SECONDS)
+                .waitForSuccess(5, TimeUnit.SECONDS)
         )
         check(
             ProcessBuilder("git", "add", ".")
                 .directory(tempDir)
                 .start()
-                .waitFor(5, TimeUnit.SECONDS)
+                .waitForSuccess(5, TimeUnit.SECONDS)
+        )
+        check(
+            ProcessBuilder("git", "config", "user.name", "\"Bruce Wayne\"")
+                .directory(tempDir)
+                .start()
+                .waitForSuccess(5, TimeUnit.SECONDS)
+        )
+        check(
+            ProcessBuilder("git", "config", "user.email", "\"bruce@wayne.com\"")
```

Comment:
This might actually be a real adress… https://wayne.com/
Maybe let's use something a bit more generic (john smith @ example .com) for a public repository, even if it's less fun

Reply:
done